packages/scripts/src/main.ts

```diff
@@ -0,0 +1,21 @@
+import Vue from 'vue'
+import App from './App.vue'
+import router from './router'
+import store from './store/store'
+import BootstrapVue from 'bootstrap-vue'
+
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faEdit, faTrash, faPlay } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+
+Vue.config.productionTip = false
+
+library.add(faEdit, faTrash, faPlay)
+Vue.component('font-awesome-icon', FontAwesomeIcon)
+Vue.use(BootstrapVue)
```

Comment:
have you considered not using bootstrap vue ? :)

Reply:
Yes i have, still going to use it though. 😋